### DISCUSSION TITLE
人脸识别问题

### DISCUSSION THREAD

**Discussion by yang1158280920** [exadel-inc/CompreFace#682] [Q&A]
有没有API  通过给照片返回对应类似的 image_id

**Comment by pospielov** [comment]
https://github.com/exadel-inc/CompreFace/blob/master/docs/Rest-API-description.md#recognize-faces-from-a-given-image
I think this is what you need.

**Reply by yang1158280920** [reply]
I need to get the image_id from the photo

**Reply by pospielov** [reply]
Got it, I think no, now CompreFace returns Subject, not image_id
On another hand, if you have one image per subject, you can get the subject and then use this endpoint to get the image_id:
https://github.com/exadel-inc/CompreFace/blob/master/docs/Rest-API-description.md#list-of-all-saved-examples-of-the-subject

**Reply by yang1158280920** [reply]
thanks
